crates/superposition_types/src/webhook.rs

```diff
@@ -43,9 +50,17 @@ pub enum WebhookEvent {
     ExperimentConcluded,
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct EventInfo {
+    pub webhook_event: WebhookEvent,
+    pub time: NaiveDateTime,
```

Comment:
Why not string, then you don't need to import chrono

Reply:
Why string?
importing chrono will cause any issue?

Reply:
Yeah superposition_types is used in all other places, so it increases the size, compile time, etc. If we can avoid it, since it's used only to store the timestamp which is a string at the end - then fine